.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: java
 install: true
-script: mvn -Pfast-tests -Pextra-tests -B install
```

Comment:
smoke-tests are depending on the install, as they will create the broker from the artemis-home under the distribution
if you remove this, those tests will not build.. as the build is failing for you now.

Reply:
Perhaps you could add a comment (don't know what's the format for comments on yml) documenting why the install is needed?